docs/source-app/levels/basic/hello_components/deploy_model.py

```diff
@@ -0,0 +1,15 @@
+# A hello world component
```

Comment:
I think we had some problem with having py files/examples in docs so we moved them to examples and with docs build time copied them...